app/controllers/application_controller.rb

```diff
@@ -1,5 +1,7 @@
 class ApplicationController < ActionController::Base
   def current_user
-    User.first
+    @current_user ||= User.first
```

Comment:
- [ ] You have not used `@current_user` anywhere, so kindly replace it with `User.first` as everywhere its calling `current_user` method only.